index.html

```diff
@@ -0,0 +1,22 @@
+<!doctype html>
+<head>
+    <meta charset="utf-8">
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3" crossorigin="anonymous">
+</head>
+<body onload="getUsers()">
+    <div class="container">
+        <table id="table" class="table">
+        </table>
+    </div>
+
+    <hr>
+    <div class="container form-label">
```

Comment:
I had to scroll over 100 rows down to be able to get to this form. Not the best solution. Better to move it on top of page. But this is minor thing.